profiler/src/main/java/com/splunk/opentelemetry/logs/CommonAdapter.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.logs;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.proto.common.v1.AnyValue;
+import io.opentelemetry.proto.common.v1.ArrayValue;
+import io.opentelemetry.proto.common.v1.InstrumentationLibrary;
+import io.opentelemetry.proto.common.v1.KeyValue;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import java.util.List;
+
+/** Copied from otel internals, because it's so locked down we can't use it here. */
+final class CommonAdapter {
+  @SuppressWarnings("unchecked")
+  public static KeyValue toProtoAttribute(AttributeKey<?> key, Object value) {
+    KeyValue.Builder builder = KeyValue.newBuilder().setKey(key.getKey());
```

Comment:
`builder` is not used in the switch statement at all; how about moving it just before the last `return`?

Reply:
Sure. I copied it outright and didn't add any tests nor did I attempt any cleanup.  :)